src/main/scala/nl.knaw.dans.easy.dd2d/mapping/BlockDataVaultMetadata.scala

```diff
@@ -20,8 +20,8 @@ trait BlockDataVaultMetadata {
   val DATAVERSE_PID_VERSION = "dansDataversePidVersion"
   val BAG_ID = "dansBagId"
   val NBN = "dansNbn"
-  val OTHER_ID = "dansOtherId"
-  val OTHER_ID_VERSION = "dansOtherIdVersion"
+  val DANS_OTHER_ID = "dansOtherId"
```

Comment:
I added it because also in the `BlockCitation `we have `OTHER_ID`. `DdmToDataverseMapper` is extended by both, `BlockCitation` and `BlockDataVaultMetadata`.

Reply:
OK, that seems like a good workaround.